packages/bechamel-perf/bechamel-perf.0.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/bechamel"
+bug-reports:  "https://github.com/dinosaure/bechamel/issues"
+dev-repo:     "git+https://github.com/dinosaure/bechamel.git"
+doc:          "https://dinosaure.github.io/bechamel/"
+license:      "MIT"
+synopsis:     "Linux perf's metrics for bechamel"
+description: """A simple layer on Linux perf's metrics for
+bechamel to record and analyze them."""
+
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml"      {>= "4.08.0"}
+  "dune"       {>= "2.0.0"}
+  "mperf"
```

Comment:
Shouldn't this be also bound to the same version as the package?

Reply:
Hmmhmm, `mperf` will be not updated for a long time - so it should not follow the same release cycle of `bechamel`.

Reply:
ah my bad, I'll revert my change.